src/tvu/repair/service.zig

```diff
@@ -0,0 +1,436 @@
+const std = @import("std");
+const zig_network = @import("zig-network");
+const sig = @import("../../lib.zig");
+
+const Allocator = std.mem.Allocator;
+const Atomic = std.atomic.Atomic;
+const KeyPair = std.crypto.sign.Ed25519.KeyPair;
+const Random = std.rand.Random;
+const Socket = zig_network.Socket;
+
+const ContactInfo = sig.gossip.ContactInfo;
+const GossipTable = sig.gossip.GossipTable;
+const Logger = sig.trace.Logger;
+const LruCacheCustom = sig.common.LruCacheCustom;
+const Nonce = sig.core.Nonce;
+const Pubkey = sig.core.Pubkey;
+const RwMux = sig.sync.RwMux;
+const SocketAddr = sig.net.SocketAddr;
+const Slot = sig.core.Slot;
+
+const RepairRequest = sig.tvu.repair.RepairRequest;
+const serializeRepairRequest = sig.tvu.repair.serializeRepairRequest;
+
+fn getLatestSlot() u64 {
+    return 255043752;
+}
+
+/// Identifies which repairs are needed and sends them
+/// - delegates to RepairPeerProvider to identify repair peers.
+/// - delegates to RepairRequester to send the requests.
+pub const RepairService = struct {
+    allocator: Allocator,
+    requester: RepairRequester,
+    peers: RepairPeerProvider,
```

Comment:
nit: change field name to `peer_provider`

Reply:
all renames in 1 commit: https://github.com/Syndica/sig/pull/87/commits/4e02024cfbff9143db5fdf9ec08dbb32a2cef83a